.gitignore

```diff
@@ -0,0 +1,3 @@
+/composer.lock
+/vendor/
+/Packages/
```

Comment:
These folders would not be present in this packages folder - only in the root of a distribution

Reply:
We could install the flow base distribution and require this package via the $BRANCH variable. I can look it up from another package I know of?

Reply:
As long as we do not need it i like this approach better as it is very easy to maintain and we use this in other packages aswell. Since functional tests require credentials which we will not enter in github we cannot run them anyways. 

Having barebones unit tests has the advantage of beeing really fast.

Do you see any problem with the little .gitignore in there?

Reply:
> Do you see any problem with the little .gitignore in there?

It's the idea about excluding things that should never be present in the first pace :)